viewPost.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+include('config/init.php');
+
+
+$result=getAllBlogPosts();
+$blogPosts= getBlogPost($_GET['categoryId']);
```

Comment:
This is another pluralization mismatch. The function name suggests it's getting one blog post, but the variable name suggests it's an array